pkg/api/status.go

```diff
@@ -17,7 +17,7 @@ import (
 )
 
 type statusSnapshotResponse struct {
-	Peer                    string  `json:"peer"`
+	Overlay                 string  `json:"overlay"`
```

Comment:
Should we bring this one up for discussion? I haven't had a chance to talk it over with anyone else.